starter-code/routes/drones.js

```diff
@@ -1,12 +1,18 @@
 const express = require('express');
-
-// require the Drone model here
+const Drone = require("../models/drone").Drone;
 
 const router = express.Router();
 
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2
+router.get('/', (req, res, next) => {
+  Drone.find({}, (err, drones) => {
+    if(err){
+      console.log(err);
+    }
```

Comment:
Do this instead:

always create a data object with key and value:

```
router.get('/', (req, res, next) => {
    Drone.find({}, (err, drones) => {
        if (err) {
            next(err);
        }
        const data = {
            drones: drones,
        }
        res.render('./drones', data);
    });
});
```

Reply:
res.render('drones', data); 
"drones" is the folder and in the drones folder you create a file called index.ejs